### DISCUSSION TITLE
Welcome to LibreChat Discussions!

### DISCUSSION THREAD

**Discussion by danny-avila** [danny-avila/LibreChat#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stepantu** [comment]
I can’t enter the key, how to do this please help

**Comment by wenygit** [comment]
What is meilisearch for?

**Reply by fuegovic** [reply]
@wenygit it's for the search feature
see: https://www.librechat.ai/docs/user_guides/search

![image](https://github.com/danny-avila/LibreChat/assets/32828263/b6561c90-fc16-4002-845b-9bb47b0c5eca)

**Comment by vislamu** [comment]
Hi If I use Dalle-3 to generate images, how did the token usage? Where can I set it or make some modify?

**Comment by jumpei-kurata** [comment]
[Enhancement: Chat rating button #901](https://github.com/danny-avila/LibreChat/issues/901)
I am about to embark on this task. However, I am tripping up on the environment building part.
When I do npm run backend:dev, I am getting the following error.
I am trying to develop using devcontainer on a Mac VSCode.
Do I need to have vectordb settings etc in docker-compose.yml?

```
2024-09-16 03:31:36 info: [Optional] Redis not initialized. Note: Redis support is experimental.
2024-09-16 03:32:14 error: KeyvMongo connection error: connection timed out
2024-09-16 03:32:14 error: There was an uncaught error: connection timed out
[nodemon] app crashed - waiting for file changes before starting...
```
